packages/server/graphql/public/types/Company.ts

```diff
@@ -0,0 +1,204 @@
+    ).filter(errorFilter)
+    const organizationUsers = organizationUsersByOrgId.flat()
+    const activeOrganizationUsers = organizationUsers.filter((organizationUser) => {
+      const isActive = !organizationUser.inactive
+      const joinedAfter = after ? organizationUser.joinedAt > new Date(after) : true
+      return isActive && joinedAfter
+    })
+    const userIds = activeOrganizationUsers.map((organizationUser) => organizationUser.userId)
+    const uniqueUserIds = new Set(userIds)
+    return uniqueUserIds.size
+  },
+  activeOrganizationCount: async ({id: domain}, _args, {dataLoader}) => {
+    const organizations = await dataLoader.get('organizationsByActiveDomain').load(domain)
+    const allOrgIds = organizations.map(({id}) => id)
+    // filter out orgs with 0 teams
+    const teamsByOrgId = (await dataLoader.get('teamsByOrgIds').loadMany(allOrgIds)).filter(
+      errorFilter
+    )
+    const teams = teamsByOrgId.flat().filter(({isArchived}) => !isArchived)
+    const orgIdsWithManyTeamsSet = new Set<string>()
+    teams.forEach((team) => {
+      orgIdsWithManyTeamsSet.add(team.orgId)
+    })
+    const orgIdsWithManyTeams = [...orgIdsWithManyTeamsSet]
+    // get the number of teams for each org
+
+    // get the number of org users for each org
+    const organizationUsersByOrgId = (
+      await dataLoader.get('organizationUsersByOrgId').loadMany(orgIdsWithManyTeams)
+    ).filter(errorFilter)
+    const organizationUsers = organizationUsersByOrgId.flat()
+    const activeOrganizationUsers = organizationUsers.filter(
+      (organizationUser) => !organizationUser.inactive
+    )
+    // filter out orgs with only 1 org user
+    const activeOrganizations = orgIdsWithManyTeams.filter((orgId) => {
+      return activeOrganizationUsers.find((organizationUser) => organizationUser.orgId === orgId)
+    })
+    return activeOrganizations.length
+  },
+  lastMetAt: async ({id: domain}, _args, {dataLoader}) => {
+    const r = await getRethink()
+    const organizations = await dataLoader.get('organizationsByActiveDomain').load(domain)
+    const orgIds = organizations.map(({id}) => id)
+    const teamsByOrgId = (await dataLoader.get('teamsByOrgIds').loadMany(orgIds)).filter(
+      errorFilter
+    )
+    const teams = teamsByOrgId.flat()
+    const teamIds = teams.map(({id}) => id)
+    if (teamIds.length === 0) return 0
+    const lastMetAt = await r
+      .table('NewMeeting')
+      .getAll(r.args(teamIds), {index: 'teamId'})
+      .max('createdAt' as any)('createdAt')
+      .default(null)
+      .run()
+    return lastMetAt
+  },
+
+  meetingCount: async ({id: domain}, {after}, {dataLoader}) => {
+    const r = await getRethink()
+    const organizations = await dataLoader.get('organizationsByActiveDomain').load(domain)
+    const orgIds = organizations.map(({id}) => id)
+    const teamsByOrgId = (await dataLoader.get('teamsByOrgIds').loadMany(orgIds)).filter(
+      errorFilter
+    )
+    const teams = teamsByOrgId.flat()
```

Comment:
+1 these lines are repeated four times. Based on our rule of three, we could move it to a helper function, but it's not a priority